src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -44,6 +44,22 @@ def load_arguments_eh(self, _):
         c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
         c.argument('default_action', arg_group='networkrule', options_list=['--default-action'], arg_type=get_enum_type(['Allow', 'Deny']),
                    help='Default Action for Network Rule Set.')
+        c.argument('zone_redundant', options_list=['--zone-redundant'], arg_type=get_three_state_flag(),
+                   help='Enabling this property creates a Standard EventHubs Namespace in regions supported availability zones')
+        c.argument('identity', arg_group='Managed Identity', options_list=['--assign-identity'], arg_type=get_three_state_flag(),
+                   help='A boolean value that indicates whether Managed Identity is enabled.')
+
+    with self.argument_context('eventhubs namespace create') as c:
+        c.argument('cluster_arm_id', options_list=['--cluster-arm-id'], help='luster ARM ID of the Namespace')
+
+    with self.argument_context('eventhubs namespace update', arg_group='Managed Identity') as c:
+        c.argument('key_source', options_list=['--key-source'], arg_type=get_enum_type(KeySource),
```

Comment:
If no specified, which kind of key source will be used in eventhub?

Reply:
Microsoft.KeyVault